### ISSUE TITLE
Pos Protocol

### ISSUE THREAD

**Issue by MYiazou** [wix/wix-restaurants-api#6]
Hi,
We are a software house from Greece named Sunsoft. 
We have a POS system that has a web based Api which receives Http requests.
Our customer has a Wix-restaurant site and our POS software.
We'd like to know if it is possible to create a specific Json format from Wix site and post the order to
our Web Service . 
According to your Api .md 
https://github.com/wix/wix-restaurants-api/blob/master/Integrating-POS-(point-of-sale)-systems-to-Wix-Restaurants.md
We want to submit orders based on 3.
Can you provide us details on how can we achive this?

**Comment by dleshem** [comment]
@MYiazou, I'll assume you're OK with handling card payments via an external payment gateway (and not internally by the POS) - otherwise the integration would be more complicated due to PCI-DSS requirements.

If that's the case, I highly recommend you use the standard webhook method (see [Using a webhook to Integrate Wix Restaurants with Third-Party Services](https://support.wix.com/en/article/using-a-webhook-to-integrate-wix-restaurants-with-third-party-services)). The webhook protocol is described [here](http://developers.wixrestaurants.com/Wix-Restaurants-POS-(point-of-sale)-SPI).

Basically, we'll HTTP POST incoming orders to a URL you set up. You can then parse the posted order data, convert it to your format and submit to your POS API.

**Comment by MYiazou** [comment]
dleshem, Thanks for your prompt response. 
We'll convert your order data and send it to our POS Api. 
There is no Quantity field to your Json format order. So if you have 2 exactly the same items you will post it twice in your Json order?
Thanks in advance

**Comment by dleshem** [comment]
See [OrderItem.count](https://github.com/wix/openrest4j/blob/e8790b31b74f400fc8528bae71b41026c6e42d43/openrest4j-api/src/main/java/com/openrest/v1_1/OrderItem.java#L117). If this property doesn't exist in the JSON it's assumed to be 1.

I'm closing this issue for now, feel free to open new specific issues if you have any questions.
